mantra/core/bot.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+import aiohttp
+import hikari
+import lightbulb
+from mantra import CACHE_SETTINGS
+from mantra.config import bot_config
+
+logger = logging.getLogger(__name__)
+
+
+class Mantra(lightbulb.BotApp):
+    def __init__(self) -> None:
+        super().__init__(
+            token=bot_config.token,
+            default_enabled_guilds=bot_config.test_guilds,
+            intents=hikari.Intents.ALL,
+            help_slash_command=True,
+            # cache_settings=CACHE_SETTINGS,
```

Comment:
You can remove comment if not needed

Reply:
This comment will be eventually added.
So it will stay there for now!
I am working on a fix to an issue this is causing.